python/meson.build

```diff
@@ -0,0 +1,10 @@
+project('ufo-python', 'c',
+        default_options : ['c_std=c99', 'warning_level=3'],
+        version : '1.1.0')
```

Comment:
@llohse is this a copy-paste from something? I would perhaps use the same version as the `ufo-core` version.

Reply:
I took it from here: https://github.com/ufo-kit/ufo-core/blob/master/python/setup.py#L9

But yes...